projects/libressl/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN git clone --depth 1 https://github.com/guidovranken/cryptofuzz
 RUN git clone --depth 1 https://github.com/guidovranken/cryptofuzz-corpora
 RUN wget https://boostorg.jfrog.io/artifactory/main/release/1.74.0/source/boost_1_74_0.tar.bz2
 RUN git clone --depth 1 https://github.com/randombit/botan.git
-RUN wget --no-check-certificate https://gmplib.org/download/gmp/gmp-6.2.1.tar.lz
+RUN wget --no-check-certificate https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.lz
```

Comment:
Do you need to bypass cert checks with gnu's mirror?

Reply:
Merging now to get the build running again, feel free to submit follow up PR if needed.

Reply:
Indeed we don't need the workaround now, thanks for pointing it out. Thanks for merging, I've made the follow-up PR: https://github.com/google/oss-fuzz/pull/10736